content/posts/April-2020.md

```diff
@@ -44,3 +44,25 @@ draft: false
 [Marshmallow3]: https://github.com/packit-service/packit-service/pull/538
 [fixed]: https://github.com/packit-service/packit-service/pull/541
 [parsing release events]: https://github.com/packit-service/packit-service/issues/536
+
+## Week 16 (April 14th - April 17th)
+
+### packit
+
+* Job metadata field `dist-git-branch` is now marked as deprecated, to be
+  replaced by `dist_git_branches` to match the naming of other metadata fields
+  and to accept multiple branch names where Packit should work. ([#797] and
+  [#788].)
+* Two new job metadata fields, `scratch` and `branch`, were introduced. These
+  will be used by Packit when building in Koji becomes available. ([#796] and
+  [#800].)
```

Comment:
@lachmanfrantisek is this correct?

Reply:
- `scratch` will be used for koji builds.
- `branch` can be used for copr builds (not announced, not documented and not turned-on in the production GitHub app settings yet) and will be used for koji builds as well.

Reply:
We discussed that it's better to drop this for now and announce them once they are available.